### PR TITLE
Side 2043 documentation

### PR DESCRIPTION
**Pull request by cshields-oliveai**

### CODE REVIEW

File: ldk/goja/typedoc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "exclude": [
+      "coverage/**/*",
+      "jest.config.js",
+      "**/*.test.ts"
+    ],
+    "listInvalidSymbolLinks": true,
+    "inputFiles": ["src/index.ts"],
+    "stripInternal": true,
+    "theme": "pages-plugin",
+    "name": "Secure Loop Development Kit for Node.js",
```
Comment (RusGraf) [inline ldk/goja/typedoc.json:17]: Should it be node.js or something else?

Comment (cshields-oliveai) [reply ldk/goja/typedoc.json:17]: Good question. How about "Javascript Loop Development Kit"?

Comment (RusGraf) [reply ldk/goja/typedoc.json:17]: Sure why not ;)

**Review by RusGraf — APPROVED** [review]
image

**Review by BrettMugglinOlive — APPROVED** [review]

**Comment by richseviora** [comment]
This doesn't appear to be working. What I think is likely the problem is that both the Goja and Node doc actions are generating files and then overwriting the contents of the `gh-pages` branch. You'll instead want to have one action that builds both doc files and then pushes it via the `actions-gh-pages` action to the one folder.

**Pull request merged**